App/SB/views/FeedList/index.js

```diff
@@ -41,7 +42,7 @@ class Notifications extends React.PureComponent {
         />
       </HeaderButtons>
     )
-    const headerTitle = 'Feed'
+    const headerTitle = 'Notifications'
```

Comment:
Sander and I talked about this briefly on slack... adding here, feel free to reject this one :)

Reply:
I like it